docs/excel/excel-add-ins-conditional-formatting.md

```diff
@@ -0,0 +1,305 @@
+const sheet = context.workbook.worksheets.getItem("Sample");
+const range = sheet.getRange("B8:E13");
+const conditionalFormat = range.conditionalFormats.add(
+     Excel.ConditionalFormatType.iconSet
+);
+
+const iconSetCF = conditionalFormat.iconSet;
+iconSetCF.style = Excel.IconSet.threeTriangles;
+
+/*
+   With a "three*” icon set style, such as "threeTriangles", the third
+    element in the criteria array (criteria[2]) defines the "top" icon;
+    e.g., a green triangle. The second (criteria[1]) defines the "middle"
+    icon, The first (criteria[0]) defines the "low" icon, but it can often 
+    be left empty as this method does below, because every cell that
+   does not match the other two criteria always gets the low icon.
+*/
+iconSetCF.criteria = [
+    {} as any,
+      {
+        type: Excel.ConditionalFormatIconRuleType.number,
+        operator: Excel.ConditionalIconCriterionOperator.greaterThanOrEqual,
+        formula: "=700"
+      },
+      {
+        type: Excel.ConditionalFormatIconRuleType.number,
+        operator: Excel.ConditionalIconCriterionOperator.greaterThanOrEqual,
+        formula: "=1000"
+      }
+];
+
+await context.sync();
+```
+
+### [Preset criteria](https://docs.microsoft.com/javascript/api/excel/excel.presetcriteriaconditionalformat)
+
+Preset formatting applies a user-defined format to the range based on a selected standard rule. These rules are defined by the [ConditionalFormatPresetCriterion](https://docs.microsoft.com/javascript/api/excel/excel.ConditionalFormatPresetCriterion) in the [ConditionalPresetCriteriaRule](https://docs.microsoft.com/javascript/api/excel/excel.conditionalpresetcriteriarule). The following example colors the font white wherever a cell’s value is at least one standard deviation above the range’s average.
+
+![A range with white font cells where the values are at least one standard deviation above average.](../images/excel-conditional-format-preset.png)
+
+```typescript
+const sheet = context.workbook.worksheets.getItem("Sample");
+const range = sheet.getRange("B2:M5");
+const conditionalFormat = range.conditionalFormats.add(
+     Excel.ConditionalFormatType.presetCriteria
+);
+
+// color every cell’s font white that is one standard deviation above average relative to the range
+conditionalFormat.preset.format.font.color = "white";
+conditionalFormat.preset.rule = {
+     criterion: Excel.ConditionalFormatPresetCriterion.oneStdDevAboveAverage
+};
+
+await context.sync();
+```
+
+### [Text comparison](https://docs.microsoft.com/javascript/api/excel/excel.textconditionalformat)
+
+Text comparison formatting uses string comparisons as the condition. The `rule` property is a [ConditionalTextComparisonRule](https://docs.microsoft.com/javascript/api/excel/excel.conditionaltextcomparisonrule) defining a string to compare with the cell and an operator to specify the type of comparison. The following example formats the font color red when a cell’s text contains “Delayed”.
```

Comment:
To be consistent with the opening sentence of other like paragraphs, consider starting this one with "Text comparison conditional formatting..."